src/widget/scroll.rs

```diff
@@ -61,6 +84,20 @@ impl<T: Data> Scroll<T> {
             false
         }
     }
+
+    /// Limit scroll behavior to allow only vertical scrolling (Y-axis).
+    /// The child is laid out with width constrained layout bounds.
```

Comment:
maybe, "The child is laid out with constrained width and infinite height."?

if you prefer this you can make the same change below.

Reply:
changed